developers.libra.org/CONTRIBUTING.md

```diff
@@ -1,52 +0,0 @@
-# Contribution Guide
```

Comment:
I'm assuming that all of these points are already covered in the top-level one.

Reply:
Yes, this is actually a really weird doc that it looks like it was intended for libra code and not website

Reply:
Whoa. I wonder if I messed something up and didn't get the right doc in this directory. Let me check.

Reply:
No. This is the way it was on the `website` repo. So, yeah, we should remove this and maybe put a new `CONTRIBUTING.md` in `developers.libra.org` that talks about how to specifically contribute and test the docs. But we can do that later.